zenoh/src/net/routing/interceptor/authz.rs

```diff
@@ -0,0 +1,153 @@
+use std::fmt;
+
+// use casbin::{CoreApi, Enforcer};
+use super::RoutingContext;
+use casbin::prelude::*;
+use zenoh_keyexpr::keyexpr_tree::box_tree::KeBoxTree;
+use zenoh_protocol::network::NetworkMessage;
+use zenoh_result::ZResult;
+
+pub enum Action {
+    NONE,
```

Comment:
Enum variants don't follow naming convention in Rust. Make sure to include this file in some module and to run `cargo clippy`.